drivers/gpio/gpio_nct38xx_port.c

```diff
@@ -70,7 +69,7 @@ static int gpio_nct38xx_pin_config(const struct device *dev, gpio_pin_t pin, gpi
 		}
 
 		new_reg = reg | mask;
-		/* NCT3807 bit3 should be set to 0 */
```

Comment:
probably should be a separate commit from the gpio dir + config commit

Reply:
It is separate commit, just the same PR